app/views/case_contacts/form/details.html.erb

```diff
@@ -79,6 +79,24 @@
       </div>
     </div>
 
+    <div class="card-style-1 pl-25 mb-10">
+      <h4 class="mb-20"><label for="case_contact_casa_case">1. Select the relevant <%= "case".pluralize(@casa_cases.count) %> for this contact <span class="red-letter">*</span></label></h4>
+      <div class="">
+        <% @case_contact.contact_topic_answers.each do |topic| %>
+          <div class="form-check checkbox-style mb-10 ml-5">
+            <%= check_box_tag "case_contact[draft_case_ids][]",
+              0,
+              topic.selected,
+              id: "case_contact_draft_case_id_#{0}",
+              class: ["form-check-input", "casa-case-id"] %>
+            <label class="form-check-label" for="case_contact_draft_case_id_0">
```

Comment:
The ids in `id` and `for` should be tied to the topic's id. Currently, this happens.
https://www.loom.com/share/5cd50cc412d847e9ac8a8736c308fdbf?sid=388a37ce-b953-43d7-9491-ca8bc63e6a47

Reply:
Line 90 is causing a serious bug.
https://www.loom.com/share/92d7526db848427c87e56998383a5347?sid=fab1458c-7648-446e-b7f0-7dbbe78d7202